index.html

```diff
@@ -0,0 +1,353 @@
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+
+      <div>
+        <section id="contact" class="contact">
+          <h2 class="section-title">Contact me</h2>
+          <span class="section-subtitle">Get in touch</span>
+  
+          <p>
+            If you have an application that you are interested in developing, a
+            feature that you need to be built, or a project that needs coding, I'd
+            love to help with it.
+          </p>
+  
+          <form
+            action="https://formspree.io/f/xeqnevwb"
+            method="post"
+            id="form"
+            class="form"
+          >
+            <label for="name"></label>
+            <input
+              class="contact-input"
+              id="name"
+              type="text"
+              maxlength="30"
+              required
+              name="fullName"
+              placeholder="Enter your Full name"
+            />
+  
+            <label for="mail"></label>
+            <input
+              class="contact-input"
+              id="email"
+              type="email"
+              required
+              name="userEmail"
+            />
+  
+            <label for="message"></label>
+            <textarea
+              class="contact-textarea"
+              name="userMessage"
+              id="message"
+              required
+              placeholder="Write your message here"
+            ></textarea>
```

Comment:
- [ ] Please add the `maxlength="500"` attribute in your `textarea` as per project requirement, so that a user can send their queries in short and descriptive text. 
- Kindly look at the below snippet.

![Screenshot from 2023-04-20 01-20-41](https://user-images.githubusercontent.com/66476329/233184699-4cbff549-7ff7-4b2a-bea6-33cb454886b2.png)